openwisp_controller/config/admin.py

```diff
@@ -611,7 +611,7 @@ class VpnAdmin(MultitenantAdminMixin, BaseConfigAdmin, UUIDAdmin):
     ]
 
     class Media(BaseConfigAdmin):
-        js = list(BaseConfigAdmin.Media.js) + [static(f'{prefix}js/vpn.js')]
```

Comment:
Since we won't need to wait for the `DH` to generated, this won't be needed anymore. So I removed it